### ISSUE TITLE
Error: Cannot read properties of undefined (reading 'has')

### ISSUE THREAD

**Issue by shacky** [Linkerin/starlight-to-pdf#1]
```
# nvm use
Found '/home/shacky/.nvmrc' with version <lts/hydrogen>
Now using node v18.20.3 (npm v10.7.0)

# npx starlight-to-pdf https://help.contractors.es/
[INFO]: Welcome to Starlight to PDF tool! 📖
[INFO]: version: null

[INFO]: Looking for next page. Navigating to https://help.contractors.es/pl#_top...
[INFO]: Looking for next page. Navigating to https://help.contractors.es/pl...
[INFO]: Looking for next page. Navigating to https://help.contractors.es/pl/first-setup/1/...
[INFO]: Docs content was found. Starting page: https://help.contractors.es/pl/first-setup/1/

[ERROR]: Unknown error occurred.
[ERROR]: Error: Cannot read properties of undefined (reading 'has')
[ERROR]: Program exits.
```

**Comment by Linkerin** [comment]
Thank you for the issue. Fixed in version 0.0.11. Please, check for your use case.

**Comment by shacky** [comment]
Works fine! Great, thank you!
